components/web/squid/Makefile

```diff
@@ -95,50 +95,44 @@ CONFIGURE_OPTIONS += --enable-referer-log
 CONFIGURE_OPTIONS += --enable-removal-policies='heap,lru'
 CONFIGURE_OPTIONS += --enable-snmp
 CONFIGURE_OPTIONS += --enable-ssl
+CONFIGURE_OPTIONS += --enable-ssl-crtd
+#CONFIGURE_OPTIONS += --enable-devpoll
+CONFIGURE_OPTIONS += --enable-zph-qos
+CONFIGURE_OPTIONS += --enable-icap-client
+CONFIGURE_OPTIONS += --with-openssl=$(OPENSSL_PREFIX)
 CONFIGURE_OPTIONS += --enable-storeio='aufs,diskd,ufs'
+CONFIGURE_OPTIONS += --enable-storeid-rewrite-helpers=file
+CONFIGURE_OPTIONS += --enable-inline
 CONFIGURE_OPTIONS += --enable-useragent-log
+CONFIGURE_OPTIONS += --enable-log-daemon-helpers='DB,file'
 CONFIGURE_OPTIONS += --enable-x-accelerator-vary
+CONFIGURE_OPTIONS += --enable-translation
 CONFIGURE_OPTIONS += --with-aio
 CONFIGURE_OPTIONS += --with-aufs-threads=8
 CONFIGURE_OPTIONS += --with-large-files
-CONFIGURE_OPTIONS += --with-build-environment=POSIX_V6_ILP32_OFFBIG
+CONFIGURE_OPTIONS += --with-build-environment=POSIX_V6_LP64_OFF64
 CONFIGURE_OPTIONS += --with-pthreads
+# the following option is not documented but without kerberos does not work
+CONFIGURE_OPTIONS += --with-solaris-krb5
+CONFIGURE_OPTIONS += --enable-static=no
+CONFIGURE_OPTIONS += --without-heimdal-krb5
+CONFIGURE_OPTIONS += --without-mit-krb5
+CONFIGURE_OPTIONS += --without-gnugss
+CONFIGURE_OPTIONS += --without-systemd
+CONFIGURE_OPTIONS += --without-gnutls
+# These are needed for Cisco integration
+CONFIGURE_OPTIONS += --enable-wccp
+CONFIGURE_OPTIONS += --enable-wccpv2
+# Native instructions need to be disabled for zones as reported in #5221
+# This is described in detail in
+#   http://wiki.squid-cache.org/KnowledgeBase/IllegalInstructionError
+CONFIGURE_OPTIONS += --disable-arch-native
 
 # common targets
 build:		$(BUILD_64) 
 
 install:	$(INSTALL_64) 
 
```

Comment:
The common targets (build:, install:, and test:) can be removed  as they are automatically set to these values.

Reply:
For common Makefiles, you can run 'gmake update' in the component directory to apply some formatting and bump the component revision. I tried to implement the most common changes that we had to do in the past 2 years.

Reply:
Thanks Aurélien, this make target was new to me. Updated Makefile accordingly.